content/docs/rfcs/36/README.md

```diff
@@ -0,0 +1,738 @@
+
+```
+/{application-name}/{version-of-the-application}/{content-topic-name}/{encoding}
+```
+
+### `extern char* waku_pubsub_topic(char* name, char* encoding)`
+
+Create a pubsub topic string according to [RFC 23](https://rfc.vac.dev/spec/23/).
+
+**Parameters**
+
+1. `char* name`
+2. `char* encoding`: depending on the payload, use `proto`, `rlp` or `rfc26`
+
+**Returns**
+
+`char *` containing a content topic formatted according to [RFC 23](https://rfc.vac.dev/spec/23/).
+
+```
+/waku/2/{topic-name}/{encoding}
+```
+
+### `extern char* waku_default_pubsub_topic()`
+
+Returns the default pubsub topic used for exchanging waku messages defined in [RFC 10](https://rfc.vac.dev/spec/10/).
+
+**Returns**
+
+`char *` containing the default pubsub topic:
+
+```
+/waku/2/default-waku/proto
+```
+
+### `extern char* waku_relay_publish(int nodeId, char* messageJson, char* pubsubTopic, int timeoutMs)`
+
+Publish a message using waku relay.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+2. `char* messageJson`: JSON string containing the [Waku Message](https://rfc.vac.dev/spec/14/) as [`JsonMessage`](#JsonMessage).
+3. `char* pubsubTopic`: pubsub topic on which to publish the message.
+   If `NULL`, it uses the default pubsub topic.
+4. `int timeoutMs`: Timeout value in milliseconds to execute the call.
+   If the function execution takes longer than this value,
+   the execution will be canceled and an error returned.
+   Use `0` for no timeout.
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains the message ID.
+
+### `extern char* waku_relay_enough_peers(int nodeId, char* pubsubTopic)`
+
+Determine if there are enough peers to publish a message on a given pubsub topic.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+2. `char* pubsubTopic`: Pubsub topic to verify.
+   If `NULL`, it verifies the number of peers in the default pubsub topic.
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains a `boolean` indicating whether there are enough peers.
+
+For example:
+
+```json
+{
+  "result": true
+}
+```
+
+### `extern char* waku_relay_subscribe(int nodeId, char* pubsubTopic)`
+
+Subscribe to a Waku Relay pubsub topic to receive messages.
+
+**Parameters**
+
+1. `int nodeId`: The node identifier obtained from a successful execution of [`waku_new`](#extern-char-waku_newchar-configjson).
+2. `char* pubsubTopic`: Pubsub topic to subscribe to. 
+   If `NULL`, it subscribes to the default pubsub topic.
+
+**Returns**
+
+A [`JsonResponse`](#jsonresponse).
+If the execution is successful, the `result` field contains an `integer` that is the subscription id.
```

Comment:
What is the value of having a subscription ID per pubsubTopic subscription, if the expected behaviour is to receive message events already identifiable by only that pubsubTopic?

Reply:
@richard-ramos have you found this subscription id useful? Would you consider remove it and identifying a subscription by the pubsub topic instead?

Reply:
Which would mean removing `waku_relay_close_subscription` (renamed to `waku_relay_unsubscribe`) and use `waku_relay_unsubscribe_from_topic` instead.

Would that makes sense?

Reply:
@richard-ramos What does the function return ? your comment still mentions a subscription ID. Would you mind update the RFC for these two functions?